metadata-utils/src/main/java/com/linkedin/metadata/utils/GenericRecordUtils.java

```diff
@@ -98,6 +116,39 @@ public static GenericAspect serializeAspect(@Nonnull JsonNode json) {
     return genericAspect;
   }
 
+  @Nonnull
+  public static GenericAspect serializePatch(@Nonnull JsonPatch jsonPatch) {
+    GenericAspect genericAspect = new GenericAspect();
+    genericAspect.setValue(
+        ByteString.unsafeWrap(jsonPatch.toString().getBytes(StandardCharsets.UTF_8)));
+    genericAspect.setContentType(GenericRecordUtils.JSON_PATCH);
+    return genericAspect;
+  }
+
+  @Nonnull
+  public static GenericAspect serializePatch(@Nonnull JsonNode jsonPatch) {
+    GenericAspect genericAspect = new GenericAspect();
```

Comment:
Had to do a double take on how this wasn't totally duplicated heh, could probably pull a trick with generics to not duplicate the code (not needed though).

Reply:
Yeah, in this case I'd have to end up with instanceof checks to check the content-type. Or have a map or something. I figured this is a tiny method to add here.